modules/ROOT/pages/push/proc_sending-a-push-notification.adoc

```diff
@@ -0,0 +1,40 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: proc_sending-a-push-notification
+
+[id='{context}_sending-a-push-notification']
+
+= Sending a Push Notification
+
+The {push-ui} allows you to send {push-notification}s to devices.
+
+[discrete]
+== Prerequisites
+
+* Access to the {push-ui}.
+* An application with at least one binding or variant.
+
+[discrete]
+== Procedure
```

Comment:
I've stripped this down, do review and make sure I haven't lost anything important